xray/commands/audit/nuget/nuget.go

```diff
@@ -0,0 +1,67 @@
+package nuget
+
+import (
+	"os"
+
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils/dotnet/solution"
+	"github.com/jfrog/jfrog-cli-core/v2/xray/commands/audit"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+)
+
+const (
+	nugetPackageTypeIdentifier = "nuget://"
+)
+
+type AuditNugetCommand struct {
+	audit.AuditCommand
+}
+
+func NewEmptyAuditNugetCommand() *AuditNugetCommand {
+	return &AuditNugetCommand{AuditCommand: *audit.NewAuditCommand()}
+}
+
+func NewAuditNugetCommand(auditCmd audit.AuditCommand) *AuditNugetCommand {
+	return &AuditNugetCommand{AuditCommand: auditCmd}
+}
+
+func (anc *AuditNugetCommand) Run() error {
+	dependencyTree, err := anc.buildNugetDependencyTree()
+	if err != nil {
+		return err
+	}
+	return anc.ScanDependencyTree(dependencyTree)
+}
+
+func (anc *AuditNugetCommand) buildNugetDependencyTree() (nodes []*services.GraphNode, err error) {
```

Comment:
This function need to be unit tested. See that we do the same for Go in the ```TestBuildGoDependencyList``` function. It looks like the dependencies tree building is being tested similarly for all supported package managers. Let's add the same for NuGet.